### PR TITLE
change layer circle colour

### LINKED ISSUE #72: Display blue dots for parking lots at a lower zoom level
Right now if you zoom out the dots are black, even though the icons are blue:

Screenshot from 2020-02-27 11-59-07

They should be blue though.

### PR DESCRIPTION
**Pull request by Beck-berry**
Fixes #72

### CODE REVIEW

File: sass/themes/hb/_theme.scss [hunk 1]
```diff
@@ -15,3 +15,7 @@ $desktop-title-color: $primary-color;
 $bus-color: red;
 $metro-color: blue;
 $rail-color: green;
+
+.leaflet-marker-icon {
```
Comment (leonardehrenfried) [inline sass/themes/hb/_theme.scss]: And this only effects the parking lots or other layers as well?

Comment (Beck-berry) [reply sass/themes/hb/_theme.scss]: All geojson layers.

Comment (leonardehrenfried) [reply sass/themes/hb/_theme.scss]: But I just wanted the parking lots to be blue, not all of them. The geojson layers have black icons, isn;t it?

**Comment by leonardehrenfried** [comment]
This looks good but having looked at the other layers, I think we need to revisit this as the dot and the icon should have the same colour. I will leave that to @hbruch though.

**Pull request merged**